src/code/utilities/genetics-utils.js

```diff
@@ -471,4 +471,59 @@ export default class GeneticsUtils {
     });
     return {geneStart, chromosomeHeight: species.chromosomesLength[chromosomeName]};
   }
+
+  /**
+   * Given a gene string, returns an object representing the phenotypes expressed by that gene string. For example,
+   * 'a:w,b:w' is converted to {'wings': 'No wings'}.
+   */
+  static convertGeneStringToPhenotype(genes) {
```

Comment:
Is there a reason we have to specify baskets using alleles, and not traits?

It seems like making it traits instead would make everything a lot simpler.